google-built-opentelemetry-collector/Dockerfile.build

```diff
@@ -36,5 +36,5 @@ COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certifica
 COPY --from=build --chmod=755 /google-built-opentelemetry-collector/otelcol-google /otelcol-google
 COPY --from=build --chmod=644 /google-built-opentelemetry-collector/config.yaml /etc/otelcol-google/config.yaml
 
-ENTRYPOINT ["/otelcol-google", "--feature-gates=exporter.googlemanagedprometheus.intToDouble"]
+ENTRYPOINT ["/otelcol-google", "--feature-gates=exporter.googlemanagedprometheus.intToDouble,exporter.googlecloud.CustomMonitoredResources"]
```

Comment:
Same as previous nit

Reply:
This is generated using distrogen, also the docs say no spaces between them: https://github.com/open-telemetry/opentelemetry-collector/tree/main/featuregate#controlling-gates